test/functional/page_objects/settings_page.ts

```diff
@@ -305,6 +305,13 @@ export function SettingsPageProvider({ getService, getPageObjects }: FtrProvider
       );
     }
 
+    async getAllIndexPatternNames() {
+      const indexPatterns = await this.getIndexPatternList();
+      return await mapAsync(indexPatterns, async (index) => {
```

Comment:
I followed the prior work there. Is there any particular reason not to use Bluebird since the rest of the code does? Is this something that is being removed? @alisonelizabeth

Reply:
Just wanted to point out that it is not necessary to use a library for this. However, if the rest of the code is using bluebird, I'm OK with staying consistent.